scripts/watchdog_worker_timeouts.py

```diff
@@ -52,24 +38,23 @@ def tag_timedout_workers(url, timeout):
 
     # tag each with timedout
     for res in results:
-        id = res["_id"]
+        _id = res["_id"]
+        _index = res["_index"]
         src = res.get("_source", {})
-        status = src["status"]
+        # status = src["status"]
         tags = src.get("tags", [])
 
         if "timedout" not in tags:
             tags.append("timedout")
             new_doc = {"doc": {"tags": tags}, "doc_as_upsert": True}
-            response = job_utils.update_es(id, new_doc, index="worker_status-current")
+            response = job_utils.update_es(_id, new_doc, index=_index)
             if response["result"].strip() != "updated":
-                err_str = "Failed to update status for {} : {}".format(
-                    id, json.dumps(response, indent=2)
-                )
+                err_str = "Failed to update status for {} : {}".format(_id, json.dumps(response, indent=2))
                 logging.error(err_str)
                 raise Exception(err_str)
-            logging.info("Tagged %s as timedout." % id)
+            logging.info("Tagged %s as timedout." % _id)
         else:
-            logging.info("%s already tagged as timedout." % id)
+            logging.info("%s already tagged as timedout." % _id)
```

Comment:
Nice! Should we also apply a similar update to the watchdogs for jobs and tasks?

https://github.com/hysds/hysds/blob/79d579dcae0abbeb17cd87694464458b3bbf56c3/scripts/watchdog_task_timeouts.py#L53

https://github.com/hysds/hysds/blob/79d579dcae0abbeb17cd87694464458b3bbf56c3/scripts/watchdog_job_timeouts.py#L121

?